scout/adapter/mongo/case.py

```diff
@@ -686,18 +697,17 @@ def update_dynamic_gene_list(
         )
         return updated_case
 
-    def case(self, case_id=None, institute_id=None, display_name=None):
+    def case(
+        self,
+        case_id: str = None,
```

Comment:
Add Optional to the Optional params perhaps?

Reply:
Can do!